zoo-css-challenge/style.css

```
@@ -187,9 +229,13 @@ footer {
 .membership strong {
   color: #ff7f11;
 }
+.membership p {
+  max-width:700px;
+  margin: 30px 280px 30px 280px;
```

```suggestion
  margin: auto;
```
Alternatively, we can use `margin: auto` to center the element. `auto` evenly distributes the margin

---

Thank you, if I want to make more space between <p>and<button> in <section class="membership">, do I need to add "margin-bottom:  px;" in style?

---

```suggestion
  margin: 30px auto;
```

@MegumiYanaka Ah yes, if you want to have `30px` margin top and bottom like before then you can do `margin: 30px auto`, which is the shorthand way to do `30px` vertically and `auto` horizontally. Hope that makes sense